ansible/roles/operator_pipeline/templates/openshift/tasks/generate-index.yml

```diff
@@ -60,11 +60,9 @@ spec:
         # Pull the index image from a pre-configured imagestream in the current namespace.
         # It may be possible to eliminate this once sqlite-based index images are no
         # longer supported.
-        INDEX_IMG_STREAM=$(echo $(params.from_index) | rev | cut -f1 -d/ | rev)
-        FROM_INDEX="$(params.internal_registry)/$(context.taskRun.namespace)/$INDEX_IMG_STREAM"
```

Comment:
Can you please also remove the unused variable from this task? The `internal_registry` is no longer used here.

Reply:
The unused parameter is removed now.